### PR TITLE
Deployable versions of the ID conversion script and heudiconv scripts

### PR DESCRIPTION
**Pull request by willstrawson**
- Added cisc_to_adie_conversion.py - successfully converts BIDS fMRI dataset to new AIDE IDs. TODO; 1) One participant is missing, 2) need to make paths relative 3) incorporate session name change, but it works for now.
- Added heudiconv resources inside bin/heudiconv/ - hdconv_2.py + heuristic.py was used to create the fMRI BIDS dataset inside wills_data/bids

### CODE REVIEW

File: bin/cisc_to_adie_conversion.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    newdir(newid,r)
+    # F4 - Copy files from old to new structure, while renaming
+    movefiles(sub,newid,cid)
+    # F5 - Store number of files in new directory 
+    numf_new = numfiles(os.path.join(os.path.dirname(sub),newid))
+    # F6 - Check if number of files in new sub dir is == to num files in original sub dir
+    compare(numf_old,numf_new,sub)
+    # Look prettier
+    print ('\n','#####------------------- NEW SUBJECT DIRECTORY --------------------#####','\n')
+
```
Comment (htwangtw) [inline bin/cisc_to_adie_conversion.py:164]: Can you remove the trailing lines?

**Review by htwangtw — CHANGES_REQUESTED** [review]
Please modify the Path using the actual path (`/research/cisc2/.....`)on the cluster. Currently it's searched through a symbolic link to cisc2 volume to your user profile. Only you will be able to run it. 
I want you to add some explaination at the top of the script about usage example. 
Comment block in python is like this:
```python
"""
Author: So and So xx-01-2021
This script do stuff

Usage:
add how to use in the command line
"""
import os
...
```
Please change the name of the two scripts with more explicit name. What does the `2` mean? It looks like `hdconv_2.py` is built upond `hdconv.py`. 
I would name scripts as `get_metadata.py` and `run_heudiconv.py` respectively so it's clear what it does. Don't get lazy with these kind of thing.

File: bin/cisc_to_adie_conversion.py (hunk 1 above)
Comment (willstrawson) [reply bin/cisc_to_adie_conversion.py:164]: Sorry what do u mean exactly by trailing lines in this case?

**Comment by willstrawson** [comment]
Hi Hao. Here's a quick summary of the most recent changes:

- File names now make more sense
-  Added better descriptions at the top of scripts
- All scripts now have some degree of command line user input functionality. 
- All scripts now use universal paths, so anyone can run them from the SN server. I just used '/research/...' 

Importantly, i've just ran cisc_to_adie_conversion.py on all the subject dirs in wills_data/bids/ and it's converted the vast majority successfully. Hopefully these niftis will be ready to be copied into BIDS_data/ soon :) 

Thanks again for your comments as always 🥇

**Review by htwangtw — CHANGES_REQUESTED** [review]
To me, it really looks like know what still needs work! Well done!

**Pull request closed without merging**